src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -202,10 +200,10 @@ post_processing()
     return true;
 
   this->subiteration_counter=0;
-  this->anatomical_sd=0;
+  this->anatomical_sd.resize(anatomical_image_filenames.size());
 
-  if (this->anatomical_image_filenames.size()>1){
-      error("At the moment you can only use one anatomical image %s");
+  if (!this->anatomical_image_filenames.size()==sigma_m.size()){
+      error("The number of sigma_m parameters must be the same as the numberof anatomical image filenames %s");
```

Comment:
```suggestion
      error("The number of sigma_m parameters must be the same as the number of anatomical image filenames %s");
```

Reply:
however, this will crash as there is no argument to fill in the %s. if you want one, best to use `boost::format`